File: CS/TemplateCardCaptionLocalization/MyLayoutViewCustomizer.cs
using DevExpress.XtraGrid.Views.Layout.Designer;
using System.Windows.Forms;

namespace TemplateCardCaptionLocalization {
    public class MyLayoutViewCustomizer :LayoutViewCustomizer {
        protected override LayoutViewDesignerHelper CreateDesignerHelper() {
            return new MyLayoutViewDesignerHelper(this, EditingLayoutView);
        }

        protected override void CreateTemplateCardTabPage() {
            base.CreateTemplateCardTabPage();
            designerSettingsControl.Dispose();
            designerSettingsControl = new MyDesignerControlSettingsManager(this, DesignerHelper.DesignerControl);
            designerSettingsControl.Parent = designerCardSettingsScroller;
            designerSettingsControl.Dock = DockStyle.Fill;
        }
    }
}

File: VB/TemplateCardCaptionLocalization/MyLayoutViewCustomizer.vb
Imports Microsoft.VisualBasic
Imports DevExpress.XtraGrid.Views.Layout.Designer
Imports System.Windows.Forms

Namespace TemplateCardCaptionLocalization
	Public Class MyLayoutViewCustomizer
		Inherits LayoutViewCustomizer
		Protected Overrides Function CreateDesignerHelper() As LayoutViewDesignerHelper
			Return New MyLayoutViewDesignerHelper(Me, EditingLayoutView)
		End Function

		Protected Overrides Sub CreateTemplateCardTabPage()
			MyBase.CreateTemplateCardTabPage()
			designerSettingsControl.Dispose()
			designerSettingsControl = New MyDesignerControlSettingsManager(Me, DesignerHelper.DesignerControl)
			designerSettingsControl.Parent = designerCardSettingsScroller
			designerSettingsControl.Dock = DockStyle.Fill
		End Sub
	End Class
End Namespace